src/arc.ts

```diff
@@ -586,6 +599,90 @@ export function pollForAccountChanges(currentAccountAddress: Address | null, int
   });
 }
 
+async function monitorGraphNodeSubgraph() {
```

Comment:
what this function monitor?
can it be documented ?

Reply:
why it is under arc.ts file ? how about adding that under health.ts ?

Reply:
sure i can move to a different file